examples/program/BUILD.bazel

```diff
@@ -0,0 +1,16 @@
+load("@build_bazel_rules_nodejs//:defs.bzl", "jasmine_node_test")
+
+filegroup(
+    name = "node_modules",
+    srcs = glob([
+        "node_modules/**/*.js",
+        "node_modules/**/*.d.ts",
+        "node_modules/**/*.json"
+    ]),
+    visibility = ["//visibility:public"],
+)
+
+jasmine_node_test(
+    name = "program",
+    srcs = glob(["*.js"]),
```

Comment:
As a developer looking at this example, I'd like an explanation of how jasmine_node_test handles sources vs. tests, which this glob doesn't really cover.

Reply:
Good point, added a separate "library" rule as a filegroup, added that as a dep, so the srcs here are just *.spec.ts